backend/backend.js

```diff
@@ -1,142 +1,174 @@
-const express = require('express');     //Import Express Module
-const app = express();                  //Create Instance of Express
-const port = 5000;                      //Define Port Number
+const express = require('express');    
 const cors = require('cors');
 
-//export DEBUG='express:router'
-const users = { 
-    users_list :
-    [
-       { 
-          id : 'xyz789',
-          name : 'Charlie',
-          job: 'Janitor',
-       },
-       {
-          id : 'abc123', 
-          name: 'Mac',
-          job: 'Bouncer',
-       },
-       {
-          id : 'ppp222', 
-          name: 'Mac',
-          job: 'Professor',
-       }, 
-       {
-          id: 'yat999', 
-          name: 'Dee',
-          job: 'Aspring actress',
-       },
-       {
-          id: 'zap555', 
-          name: 'Dennis',
-          job: 'Bartender',
-       }
-    ]
- }
- 
+const userServices = require('./models/user-services');
+const eventServices = require('./models/events-services');
+
+const app = express();                  
+const port = 5000;          
+
+//-----------------HASH PASSWORD USING BCRYPT 10 ROUNDS----------------------------------
+
 app.use(cors());
-app.use(express.json());                //Process Data as JSON format
+app.use(express.json());               
 
-app.get('/', (req, res) => {            //API Endpoint
-    res.send('Hello World!');
+app.get('/', (req, res) => {            
+    res.send("Hello World.")
```

Comment:
maybe we can change this to a more logical response for a get request to the root directory, or delete

Reply:
I'm changing it to 'Connection to Gather Successful', I want to keep this so that if everything fails this is a default page that I can test to see if backend is up